$NetBSD: patch-lib_matplotlib.egg-info_requires.txt,v 1.2 2023/07/23 14:43:02 bacon Exp $

# Hack to allow apps to run despite known issues
# Remove this patch when the upstream issue is fixed
# https://github.com/matplotlib/matplotlib/issues/26152

--- lib/matplotlib.egg-info/requires.txt.orig	2023-07-20 13:09:04.809660522 +0000
+++ lib/matplotlib.egg-info/requires.txt
@@ -5,7 +5,7 @@ kiwisolver>=1.0.1
 numpy>=1.20
 packaging>=20.0
 pillow>=6.2.0
-pyparsing<3.1,>=2.3.1
+pyparsing>=2.3.1
 python-dateutil>=2.7
 setuptools_scm>=7
 
